README.md

```diff
@@ -1,167 +1,16 @@
-    - sub-edm010_ses-M0_PSIR.nii.gz
-    - sub-edm011_ses-M0_PSIR.nii.gz
-    - sub-edm013_ses-M0_PSIR.nii.gz
-  ```
-
-  </details>
-
-Here are several helpful videos reviewing how to correct manual lesion segmentations: [1](https://www.dropbox.com/s/j1f81vtmmmkddtv/Screen%20Recording%202023-01-09%20at%209.45.40%20AM.mov?dl=0), [2](https://www.dropbox.com/s/bm6vpcqe062t2j0/Screen%20Recording%202023-01-11%20at%201.54.36%20PM.mov?dl=0), [3](https://www.dropbox.com/s/00xjsk917wwkp7b/Screen%20Recording%202023-01-11%20at%202.20.58%20PM.mov?dl=0), [4](https://www.dropbox.com/s/3gkrfslf6gflsjg/Screen%20Recording%202023-01-11%20at%203.38.47%20PM.mov?dl=0).
-
-
-## How to use the model for MS lesion segmentation
-
-### Step 1: Cloning the Repository
-
-Open a terminal and clone the repository using the following command:
-
-~~~
-git clone https://github.com/ivadomed/canproco.git
-~~~
-
-### Step 2: Setting up the Environment
-
-The following commands show how to set up the environment. Note that the documentation assumes that the user has `conda` installed on their system. Instructions on installing `conda` can be found [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
-
-1. Create a conda environment with the following command:
-```
-conda create -n venv_nnunet python=3.9
-```
-
-2. Activate the environment with the following command:
-```
-conda activate venv_nnunet
-```
-
-3. Install the required packages with the following command:
-```
-cd canproco
-pip install -r packaging/requirements.txt
-```
-
-4. Download the model (`model_ms_seg_sc-lesion_regionBased.zip`) from the repository's latest release, which can be found [here](https://github.com/ivadomed/canproco/releases/tag/r20240125), and unzip it. 
- 
-### Step 3: Getting the Predictions
-
-To segment a single image using the trained model, run the following command from the terminal. This assumes that the model has been downloaded and is available locally. The release contains two models, the 2D nnUNet as well as the 3D nnUNet. Our experiments showed that both worked similarly. 
-
-```bash
-python packaging/run_inference_single_subject.py --path-image /path/to/image --path-out /path/to/output/directory --path-model /path/to/model 
-```
-
-The output contains the spinal cord segmentation (with value 1) and the MS lesion segmentation (with value 2). It uses a region-based approach, meaning that lesions are always located within the spinal cord segmentation. 
-
-ℹ️ The script also supports getting segmentations on a GPU. To do so, simply add the flag `--use-gpu` at the end of the above commands. By default, the inference is run on the CPU. It is useful to note that obtaining the predictions from the GPU is significantly faster than the CPU.
\ No newline at end of file
+## Table of Contents
+=================
+* [Computing Cross-Sectional Area (CSA)](scripts-t2w_csa/README.md)
+* [Spinal-cord segmentation](segment_sc_contrast-agnostic/README.md)
```

Comment:
Is this folder still relevant? We used it to obtain the initial GT SC segs, right? Then we corrected them and pushed them to git-annex. Will we ever use scripts from this folder again? If not, the folder can be deleted (it will still be available in git history if we want to go back). If we decide to keep the folder, its README needs to be updated-- the contrast-agnostic model is now part of SCT v6.2.

Reply:
Let's keep it for now, as it can be still be useful in the short future (for my work on ms-lesion-agnostic). Let's remove it later on... Do you agree ?

Reply:
okay, agree!